ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/componentutils.js

```diff
@@ -22,7 +22,7 @@
      * @param {Object} [preservedProperties] properties that will be preserved after replace
      * @return {$.Deferred} A deferred object that will be resolved when the request is completed.
      */
-    window.CQ.FormsCoreComponents.editorhooks.doReplace = function (component, editable, preservedProperties) {
+    window.CQ.FormsCoreComponents.editorhooks.doReplace = function (component, editable, preservedProperties, componentType) {
```

Comment:
I don't see the variable 'componentType' being used anywhere, is it being used for anything?